python/lsst/ts/aos/utils/m1m3/M1M3LUT.py

```diff
@@ -104,39 +112,50 @@ async def update_lut_force_balance(
         start_time,
         end_time,
     )
+    print("Elevations\n", elevations["actualPosition"].describe())
+    expected_count = interval_ms / 70.0
+    if len(elevations.index) < expected_count:
+        print(
+            f"WARNING: There is significant difference between expected {expected_count:.0f} "
+            f"and retrived {len(elevations.index):d}  - PROBABLY ELEVATION DATA ARE MISSING?"
+        )
+
     # Resample the data
     elevations = elevations["actualPosition"].resample(resample_rate).mean()
 
-    if min(elevations) > 20 and max(elevations) < 70:
+    if min(elevations) > 20 or max(elevations) < 70:
         # Add warning
         print("WARNING: Elevation data is not in the range 20-70 degrees")
 
-    print("Retrieving forces data from EFD...")
     # Get names of actuator forces
     query_forces = [f"{axis.lower()}Forces{i}" for i in range(num_actuators)]
 
     # Query appliedBalanceForces from EFD from start_time to end_time
     if force_type == ForceType.BALANCE:
+        print("Retrieving applied balance forces data from EFD...")
         forces = await client.select_time_series(
             "lsst.sal.MTM1M3.appliedBalanceForces",
             query_forces,
             start_time,
             end_time,
         )
     elif force_type == ForceType.APPLIED:
+        print("Retrieving applied forces data from EFD...")
         forces = await client.select_time_series(
             "lsst.sal.MTM1M3.appliedForces",
             query_forces,
             start_time,
             end_time,
         )
 
-        # query static forces from efd from start_time - 6h to end_time
+        # query static forces from efd from start_time - static_offset to
+        # start_time
+        print("Retrieving static forces data from EFD...")
         static_forces = await client.select_time_series(
             "lsst.sal.MTM1M3.logevent_appliedStaticForces",
             query_forces,
             start_time - TimeDelta(static_offset),
-            end_time,
+            start_time,
```

Comment:
This would yield a problem if static_offset = 0, right?

Reply:
Those lines are now gone as we are using ordering by desc..